docs/source/guide/storedata.md

```diff
@@ -0,0 +1,80 @@
+---
+title: Set up database storage
+type: guide
+order: 204
+---
+Label Studio uses a database to store project data and configuration information.
+
+## Labeling performance
+The SQLite database works well for projects with tens of thousands of labeling tasks. If you want to annotate millions of tasks or anticipate a lot of concurrent users, use a PostgreSQL database. See [Install and upgrade Label Studio](install.html#PostgreSQL-database) for more.  
+
+For example, if you import data while labeling is being performed, labeling tasks can take more than 10 seconds to load and annotations can take more than 10 seconds to perform. If you want to label more than 100,000 tasks with 5 or more concurrent users, consider using PostgreSQL or another database with Label Studio. 
+
+## SQLite database
+
+Label Studio uses SQLite by default. You don't need to configure anything. Label Studio stores all data in a single file in the specified directory of the admin user. After you [start Label Studio](start.html), the directory used is printed in the terminal. 
+
+## PostgreSQL database
+
+You can also store your tasks and completions in a [PostgreSQL database](https://www.postgresql.org/) instead of the default SQLite database. This is recommended if you intend to frequently import new labeling tasks, or plan to label hundreds of thousands of tasks or more across projects.
+
+### Create connection on startup
+
+Run the following command to launch Label Studio, configure the connection to your PostgreSQL database, scan for existing tasks, and load them into the app for labeling for a specific project.
+
+```bash
+label-studio start my_project --init --db postgresql 
+```
+
+You must set the following environment variables to connect Label Studio to PostgreSQL:
+
+```
+DJANGO_DB=default
+POSTGRE_NAME=postgres
+POSTGRE_USER=postgres
+POSTGRE_PASSWORD=
+POSTGRE_PORT=5432
+POSTGRE_HOST=db
+```
+
+### Create connection with Docker Compose
+
+When you start Label Studio using Docker Compose, you start it using a PostgreSQL database:
```

Comment:
...using Docker Compose with \[config from readme\]\(link\), you start..

Reply:
added the link to the yml that we ship in the repo, thx